package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "sass": "node-sass public/scss/main.scss public/css/main.css --output-style compressed",
     "linter": "eslint public/js/src/*js public/js/spec/*js",
-    "uglify": "uglifyjs public/js/src/ui.js public/js/src/stripe_wrapper.js -c -m -o public/js/bundle.js",
+    "uglify": "uglifyjs public/js/src/ui.js public/js/src/stripe_wrapper.js public/js/src/paypal_wrapper.js -c -m -o public/js/bundle.js",
```

Comment:
Any reason we don't uglify everything? Like `linter`?

Reply:
Why would you uglify the linter. Also, you may want to run the linter separately. For example in Travis.